webSocket/transceiver/transceiver.go

```diff
@@ -197,13 +197,14 @@ func (wt *wsTransceiver) sendAckIfNeeded(connection webSocket.WSConClient, wsMes
 }
 
 // Send will prepare and send the provided WsSendArgs
-func (wt *wsTransceiver) Send(payloadData data.PayloadData, connection webSocket.WSConClient) error {
+func (wt *wsTransceiver) Send(payloadData data.WsMessage, connection webSocket.WSConClient) error {
```

Comment:
instead of providing the data.WsMessage, you can pass the 2 needed variables: payload and operation type. It will be a little bit clearer what the function does